scripts/check_python

```diff
@@ -11,8 +11,10 @@ if hash "$PYTHON" 2>/dev/null; then
 	# Use full path from now on as it makes messages clearer
 	PYTHON=$(which $PYTHON)
 	PYTHON_VERSION=$($PYTHON -c 'import sys; print("%d.%d" % (sys.version_info[0], sys.version_info[1]))')
-	if [[ "$PYTHON_VERSION" != "2.7" ]]; then
-		echo "check-python: Python v2.7 not found (version reported by '$PYTHON' is $PYTHON_VERSION)." 2>&1
+	MINVER=2.7
+	MINVER3=3.4
```

Comment:
Is there any reason we're supporting 3.4? Instead of just 3.5? Is there value in limiting the versions of Python 3? Or on the opposite side, is there value is supporting more Python 3 versions?